style.css

```diff
@@ -0,0 +1,892 @@
+  width: 18px;
+  height: 2px;
+  background-color: #d3d3d3;
+  margin-top: 7px;
+  margin-bottom: 7px;
+}
+
+.spk p {
+  font-size: 12px;
+  color: #272a31;
+  font-weight: 520;
+}
+
+.more {
+  padding: 13px;
+  color: #272a31;
+  background-color: #fff;
+  font-weight: 500;
+  font-size: 15px;
+  margin-left: 8px;
+  border: 1px solid #272a31;
+  margin-top: 35px;
+  margin-bottom: 25px;
+  width: 305px;
+}
+
+.fa-solid {
+  color: #ec5242;
+}
+
+@media (max-width: 768px) {
+  .desktopmenu,
+  .logomenu,
+  .lastbtn,
+  .partner-2,
+  .copyrights-2,
+  #otherspeakers {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .bg-image {
+    display: none;
+  }
+
+  .more {
+    display: none;
+  }
+
+  .desktopmenu {
+    display: flex;
+    list-style-type: none;
+    background: #3c3e42;
+    color: #fff;
+    justify-content: end;
+    padding-right: 10%;
+    height: 35px;
+    align-items: center;
+  }
+
+  .desktopmenu a {
+    padding-left: 15px;
+    font-size: 13px;
+    font-size: 1vw;
+  }
+
+  .logomenu {
+    background: #fff;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    justify-items: center;
+    align-content: center;
+  }
+
+  .mainlogo {
+    align-self: start;
+    justify-self: start;
+  }
+
+  .mlogo1 {
+    justify-content: start;
+    text-decoration: none;
+    font-size: 40px;
+    font-weight: 800;
+    color: #742820;
+    cursor: pointer;
+    margin-left: 80px;
+  }
+
+  .menulist {
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+    list-style-type: none;
+  }
+
+  .menulist a {
+    list-style: none;
+    text-decoration: none;
+    color: #272a31;
+    font-size: 1.5vw;
+    padding-left: 40px;
+  }
+
+  .menulist li {
+    padding-top: 25px;
+  }
+
+  .lastbtn a {
+    color: #fff;
+    font-size: 1.5vw;
+  }
+
+  .copyrights a {
+    color: #fff;
+    font-size: 40px;
+    line-height: 40px;
+    padding-left: 50px;
+  }
+
+  .copyrights-2 a {
+    font-size: 5vw;
+    padding-left: 30px;
+    line-height: 60px;
+    padding-right: 20px;
+  }
+
+  li a:hover {
+    color: #ec5242;
+  }
+
+  .campaignBtn {
+    justify-content: end;
+    padding: 1vw;
+    color: #ec5242;
+    background-color: #fff;
+    font-weight: 500;
+    font-size: 15px;
+    margin-left: 40px;
+    border: 5px solid #ec5242;
+    margin-top: 10px;
+    margin-bottom: 5px;
+    margin-right: 9.9vw;
+  }
+
+  .intro {
+    background-image:
+      linear-gradient(
+        rgba(253, 253, 253, 0.88),
+        rgba(253, 253, 253, 0.88)
+      ),
+      url(Images/dogs.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .intro h1 {
+    font-size: 50px;
+    justify-content: start;
+    margin-left: 70px;
+    margin-top: 150px;
+    padding-bottom: 30px;
+  }
+
+  .intro-2 h1 {
+    display: none;
+  }
+
+  .intro h2 {
+    line-height: 7vh;
```

Comment:
- [ ] Kindly remove this css property so that your title cn be fully visible as it is in the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/205327778-a68ba510-d2c7-42aa-97f5-efecf3b942c6.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/205327881-6a9af233-58f0-4811-accf-67779b1e4d15.png)